arcgis-ios-sdk-samples/Maps/Generate offline map (overrides)/GenerateOfflineMapOverrides.storyboard

```diff
@@ -0,0 +1,422 @@
+                                <cells>
+                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="default" indentationWidth="10" rowHeight="96" id="17W-sk-d7w">
+                                        <rect key="frame" x="0.0" y="55.5" width="375" height="96"/>
+                                        <autoresizingMask key="autoresizingMask"/>
+                                        <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" preservesSuperviewLayoutMargins="YES" insetsLayoutMarginsFromSafeArea="NO" tableViewCell="17W-sk-d7w" id="ubR-3U-ExN">
+                                            <rect key="frame" x="0.0" y="0.0" width="375" height="95.5"/>
+                                            <autoresizingMask key="autoresizingMask"/>
+                                            <subviews>
+                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="1000" verticalHuggingPriority="1000" verticalCompressionResistancePriority="1000" text="Min Scale Level" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="eOd-Gg-WOZ">
+                                                    <rect key="frame" x="16" y="17" width="119" height="21"/>
+                                                    <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                                    <nil key="textColor"/>
+                                                    <nil key="highlightedColor"/>
+                                                </label>
+                                                <slider opaque="NO" contentMode="scaleToFill" verticalCompressionResistancePriority="1000" contentHorizontalAlignment="center" contentVerticalAlignment="center" minValue="0.0" maxValue="23" translatesAutoresizingMaskIntoConstraints="NO" id="hJV-NB-7t7">
+                                                    <rect key="frame" x="14" y="38" width="347" height="48"/>
+                                                    <connections>
+                                                        <action selector="sliderChangeAction:" destination="324-87-VxW" eventType="valueChanged" id="fel-IF-keS"/>
+                                                    </connections>
+                                                </slider>
+                                                <textField opaque="NO" contentMode="scaleToFill" enabled="NO" contentHorizontalAlignment="left" contentVerticalAlignment="center" placeholder="0" textAlignment="right" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="dQZ-Fa-xUC">
```

Comment:
Do you allow the user to enter their own value? If not, these should really be labels rather than textfields.

This is one of those `AppKit`/`UIKit` differences surprised me. I was blown away to find that there was no `NSLabel` class in `AppKit`. All labels on macOS are just textfields configured a certain way.

Reply:
Changed in the latest commit.